install/helm/open-match/values.yaml

```diff
@@ -269,6 +269,12 @@ openmatch:
     demo:
       name: openmatch-demo
       pullPolicy: Always
+    e2ematchfunction:
```

Comment:
May be a future improvement comment - but should we consider going back to the model where there is a core Open Match services installer chart - and then one for the demo / separate one for test? Lets discuss when @jeremyje is back (since he merged them into one)

Reply:
I'll open an issue to track whether we need to split up the configmap and helm chart.